backend/benefit/README.md

```diff
@@ -54,6 +54,54 @@ The project is now running at [localhost:8000](http://localhost:8000)
     * `pip-sync requirements.txt requirements-dev.txt`
 
 
+## Documentation
+
+API documentation automatically generated from OpenAPI schema can be found from [http://127.0.0.1:8000/api_docs/](http://127.0.0.1:8000/api_docs/).
```

Comment:
This url gives 404: http://127.0.0.1:8000/api_docs/

Reply:
fixed